_includes/modals.html

```diff
@@ -6,8 +6,9 @@
                 <div class="modal-header">
                     <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
                     <h3 id="dname"></h3>
+                    <span id="number_found"></span> 
                 </div>
-                <div class="modal-body" style="height:600px;overflow-y:scroll;">
+                <div class="modal-body" style="max-height:500px;overflow-y:scroll;">
```

Comment:
🤔

Reply:
The fixed height produced a long space with small query result, using a max-height would make the modal height responsive and would show a scroll bar only when the max-height is exceeded.